fonts.css

```diff
@@ -0,0 +1,27 @@
+@font-face {
+  font-family: 'PT Serif';
+  src:
+   url('fonts/PTSerif-Regular.woff2') format('woff2'),
+   url('font/PTSerif-Regular.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'PT Serif';
+  src: 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/MyBali/pullRequest?prid=5764748)